.github/actions/delete-preview/entrypoint.sh

```diff
@@ -17,7 +17,17 @@ leeway run dev/preview/previewctl:download
 
 previewctl get-credentials --gcp-service-account "${PREVIEW_ENV_DEV_SA_KEY_PATH}"
 
+if [[ -n "${INPUT_NAME:-}" ]]; then
+    TF_VAR_preview_name="${INPUT_NAME}"
+elif [[ -n "${INPUT_BRANCH:-}" ]]; then
+    TF_VAR_preview_name="$(previewctl get-name --branch "${INPUT_BRANCH}")"
+else
+    echo "Bad input arguments. One of 'name' or 'branch' has to be set."
+    exit 1
+fi
```

Comment:
What if we only have a single input, and we always pass it through `previewctl get-name --branch "${INPUT}"`

There are some cases where if a env got created, but we didn't sanitise the name, it would be an issue, but `install-context` will fail also.

I think it might simpler if we just sanitise the name always.

Reply:
I'd be happy to do that if we can find an easy way to give people the mapping from name to branch. E.g. if `previewctl list` would show the branch name too. My problem is that right now if I want to clean up a preview env manually I don't have an easy way to get the name of branch ☺️

Let's keep this PR on hold and extend it to provide such functionality (somehow)

Reply:
There's not really an easy way to do that, as we don't keep that information anywhere. The sanitised name is a one-way hash. If we want to provide that info, then we'll need to keep state, and then check against it for every env we have. I don't think it really brings any benefit, just more work for us.

> My problem is that right now if I want to clean up a preview env manually I don't have an easy way to get the name of branch

You don't need to pass the name of the branch, you can pass the sanitised name - i.e. the name of the preview env. It is equivalent to passing the non-sanitised branch name. 

In this PR if you pass a name for the preview env, and we don't sanitise it, we won't have any way to map it back when we install the context, as passing the non-sanitised one will produce a different output. If we always sanitise it, then we don't need to figure out a way to keep a map somewhere and do lookups. Win win

Reply:
We chatted about this on a call as I wasn't quite following the suggestions - we're all good now: Will change it to use `previewctl get-name --branch <input>` to sanitize the "name" (input) always. If the name is already a sanitized preview name the operation is just a no-op. If it's a branch it will be sanitized, and it already handles git refs correctly.